osu.Desktop/DiscordRichPresence.cs

```diff
@@ -152,6 +167,20 @@ private string getDetails(UserActivity activity)
             return string.Empty;
         }
 
+        private string getOnlineID(UserActivity activity)
+        {
+            switch (activity)
```

Comment:
this is way too nested, should be a single `if` rather than an `if` nested in a `switch`